src/Visualization/src/assets/sample.json

```diff
@@ -0,0 +1,10 @@
+{
+    "entries": [
+        {
+            "name": "example.js",
+            "size": 55,
+            "group": "ProjectA/package1",
+            "lineChanges": [0,0,0,0,0,10,20,25,20,20,20,20,20,0,0,0,0]
+        }
+    ]
+}
```

Comment:
I'd suggest something along like this schema for the treemap and time series animation.
Possibly we could make it hierarchic already.

Reply:
That would mean 
1) Group by time e.g. 01.01.2019-01.02.2019
2) inside this group, aggregate by file
is this correct?

About the line changes, I can already aggregate this and provide one number.
This would be lines added + lines deleted, per file, per commit, per bucket.